C/services/north-plugin-interfaces/python/python_plugin_interface.cpp

```diff
@@ -71,7 +71,6 @@ unsigned int call_plugin_send_coroutine(PyObject *plugin_send_module_func, PLUGI
 			PyObject* pReturn = PyObject_CallObject(method, arg);
 			Logger::getLogger()->info("%s:%d, pReturn=%p", __FUNCTION__, __LINE__, pReturn);
 			Py_CLEAR(arg);
-			Py_CLEAR(readingsList);
```

Comment:
Actually, it is safe to call "Py_CLEAR(readingsList)" twice, the second time would have no effect as per documentation here: https://docs.python.org/3/c-api/refcounting.html#c.Py_CLEAR
But yes, there seems to be an extra call to Py_CLEAR() in call_plugin_send_coroutine() or calling function.

Reply:
@AmandeepArora 
As per @YashTatkondawar recent tests, we see memory growth while sending data to one fledge to another one with base branch FOGL-5665, this PR has not added new issues, but it solves some crashes, mainly the ones with PyObject_Repr

Reply:
Ok. So what do you think has helped fix the issue? It doesn't seem additional Py_CLEAR would have caused an issue as per the link I sent. Is it because of removing the PyObject_Repr() code?